ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+* What is the current behavior?
```

Comment:
- Add a notes about issues on localization and that this is not a support forum.

- Is this a feature request or reporting a bug?